lib/riffed/client.ex

```diff
@@ -163,7 +163,7 @@ defmodule Riffed.Client do
       end
 
       def start_link(host, port) do
-        GenServer.start_link(__MODULE__, {host, port})
+        GenServer.start_link(__MODULE__, {host, port}, name: __MODULE__)
```

Comment:
Was this intended? 
We use this version to create pools of clients. This will make process register to the module name, effectively creating a single client. 

I think you want something like:
```elixir
def start_link(host, port, gen_server_opts \\ [])
def start_link(host, port, gen_server_opts) do 
  GenServer.start_link(__MODULE__,  {host, port}, gen_server_opts)
end
```

I can't r+ this change, it'll break pooling for everyone.